angular_components/1-base/pubspec.yaml

```diff
@@ -19,3 +19,7 @@ transformers:
     resolved_identifiers:
       Random: 'dart:math'
 - dart_to_js_script_rewriter
+
```

Comment:
Not sure we should make dartdevc the default quite yet. We'll have to make a pass to convert all the instructions away from using Dartium; that's probably when making dartdevc the default makes sense.